src/trezor.js

```diff
@@ -228,7 +228,7 @@ export class TrezorInteraction extends DirectKeystoreInteraction {
     messages.push({
       state: ACTIVE,
       level: INFO,
-      text: "Your browser should open a new Trezor Connect window.  If you do not see this window, ensure you have enabled popups for this site.",
+      text: "Your browser should now open a new window to Trezor Connect. Ensure you have enabled popups for this site.",
```

Comment:
For some reason "window to Trezor Connect" sounds odd to me.  Additionally, whose browser opens a window instead of a tab these days?

Maybe "Your browser should now open a new Trezor Connect tab..."?

Reply:
I'm also fine merging this as-is and refining the messaging later if need be.

Reply:
happy to change it to get it in, but I also think we will refine more later.

Reply:
Ah, just saw buck's comment - I think it's fine to leave it as is for now. Maybe the new designer will push us in that direction anyway.

I was trying to drop the length of the message by rewording it.